vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# variables file for ansible-coreos-setup
+logrotate: True
```

Comment:
We don't have to set it here. This var should rather be set in cluster.yml at the point where this role is invoked.